### PR TITLE
docs(integrations): add CircleCI integration example

### PR DESCRIPTION
**Pull request by rogeriopeixotocx**
**Proposed Changes**
- adding CircleCI integration example to the documentation

part of #2963 

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: docs/integrations_circleci.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+workflows:
+  version: 2
+  build:
+    jobs:
+      # etc...
+      - kics
+
+```
+
+After running a pipeline you will be able to see the Run KICS step inside the workflow details:
```
Comment (felipe-avelar) [inline docs/integrations_circleci.md:37]: After running a pipeline, you will be able to see "Run KICS" step inside workflow's details

File: docs/integrations_circleci.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+<img src="https://raw.githubusercontent.com/Checkmarx/kics/master/docs/img/circleci-build.png" width="850">
+
+Go to the artifacts tab to inspect the results:
+
+<img src="https://raw.githubusercontent.com/Checkmarx/kics/master/docs/img/circleci-artifacts.png" width="850">
+
+The results will be displayed in plain text:
```
Comment (felipe-avelar) [inline docs/integrations_circleci.md:45]: Results...

**Review by felipe-avelar — COMMENTED** [review]
Check comments, please

**Review by nunoocx — APPROVED** [review]
LGTM

**Pull request merged**